### ISSUE TITLE
Requirement: Remove blockette support

### ISSUE THREAD

**Issue by krischer** [FDSN/miniSEED3-TechnicalEvaluation#20]
Simplification of MiniSEED2: Remove blockette support.

**Comment by crotwell** [comment]
There should be a standard mapping any important information in existing blockettes into the new additional/optional data structure, #14, as required by #7

**Comment by krischer** [comment]
### Summary

(Please let me know if I missed a point or misunderstood something)

This goes together with #14 and #7. Please vote on:

1. Should we remove the abstraction of blockettes and replace it with something in #14? (Yes/No)

**Comment by crotwell** [comment]
Yes!!!

**Comment by chad-earthscope** [comment]
1. Yes.

**Comment by kaestli** [comment]
1. Yes, remove (no, don't replace with anything which is not defined contentwise)

**Comment by ozym** [comment]
1. Yes

**Comment by ketchum-usgs** [comment]
Yes,  I am assuming one of the key-value add ons would be used for further extensions of the header

**Comment by claudiodsf** [comment]
Yes!

**Comment by ihenson-bsl** [comment]
1. yes

**Comment by ValleeMartin** [comment]
1. Yes

**Comment by JoseAntonioJara** [comment]
1. Yes
